[wasm] Fix regression in compiling .bc -> .o files

radical opened the pull request:
The `-emit-llvm` arg has been incorrectly added, and removed from the
args used for compiling .bc->.o couple of times. Added a crude test to
avoid regressing in the future.

- And reduces EmccCompile's output to `Low` importance, so it doesn't pollute a normal build output (eg. for Blazor)
- And add some more tests for blazorwasm

radical committed:
[wasm] Add back --emit-llvm that got removed mistakenly, in an earlier commit

.. found thanks to Jerome Laban.

radical committed:
[wasm] Set EmccCompile's messages to MessageImportance.Low by default.

.. and to MessageImportance.Normal if `$(EmccVerbose)==true`.

radical committed:
[wasm] Quote filenames passed to emcc compile command line

radical committed:
Add more blazorwasm tests - for debug/release, aot/relinking

radical committed:
Bump sdk for workload testing to 6.0.100-rc.1.21370.2

vargaz commented on src/mono/wasm/build/WasmApp.Native.targets:
```diff
@@ -181,6 +183,7 @@
       <_EmccCFlags Include="-DCORE_BINDINGS" />
       <_EmccCFlags Include="-DGEN_PINVOKE=1" />
+      <_EmccCFlags Include="-emit-llvm" />
 
```
Won't this be passed to the .bc -> .o builds as well ?

radical replied:
So, looks like I broke this in the last few days! Adding here is fine, because it will get used only to compile the `.c` files. And the bitcode files will use the LDFlags.

I've added a crude test to check for this, so we don't regress again.

radical committed:
[wasm] Fix regression in compiling bitcode -> .o

The `-emit-llvm` arg has been incorrectly added, and removed from the
args used for compiling .bc->.o .

This commit fixes it, and adds a crude test for it, so we don't regress
again.

radical committed:
Merge remote-tracking branch 'origin/main'

radical committed:
Fix build

karelz commented:
Http3_MsQuic test failures are tracked in #56090. Should be fixed and merged shortly. Sorry for the inconvenience!
